src/visualizers/panels/ExecutionIndex/ExecutionIndexControl.js

```diff
@@ -254,7 +254,7 @@ define([
                 this._pipelineNames[desc.id] = desc.name;
             } else if (type === 'Graph') {
                 desc = this.getGraphDesc(node);
-            } else if (type === 'SubGraph') {
+            } else if (type === 'Plot2D' || type === 'Plot3D') {
```

Comment:
Can't you just check if it is a plot or SubGraph still? `client.isTypeOf` seems like it would be helpful here.

Reply:
This has been refactored to extract the `Graph` node from the children whenever possible. Since, all we care about is extracting `Graph` node.